src/components/Section.js

```diff
@@ -0,0 +1,287 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import Attributes from "./Attributes";
+import DateSection from "./DateSection";
+import Recitation from "../components/Recitation";
+import {
+  convertUnits,
+  splitLocation,
+  changeStatus,
+  styleStatus,
+  parseDate,
+} from "../utils"; // eslint-disable-line no-unused-vars
+import styled from "styled-components";
+import {
+  CalendarTodayTwoTone,
+  AddBoxTwoTone,
+  ExpandMoreOutlined,
+} from "@material-ui/icons";
+import { grey } from "@material-ui/core/colors";
+import { Collapse } from "@material-ui/core";
+// Import major progressions
+import * as actions from "../redux/modules/wishlist";
+
+function Section({
+  year,
+  semester,
+  wishlist,
+  wishlistCourse,
+  section,
+  sortedSectionMeetings,
+  courseData,
+}) {
+  const [expandedList, setExpandedList] = useState({});
+
+  const handleExpandList = (event, registrationNumber) => {
+    event.preventDefault();
+    let newLs = { ...expandedList };
+    if (registrationNumber in expandedList) {
+      newLs[registrationNumber] = !expandedList[registrationNumber];
+    } else {
+      newLs[registrationNumber] = true;
+    }
+    setExpandedList(newLs);
+  };
+
+  return (
+    <SectionContainer
+      waitlisted={
+        wishlist.filter(
+          (course) => course.registrationNumber === section.registrationNumber
+        ).length > 0
+      }
+    >
+      {courseData.name !== section.name ? (
+        <h3 className="sectionName">{section.name}</h3>
+      ) : (
+        ""
+      )}
+      {courseData.sections.length > 1 ? (
+        <h4 className="sectionNum">{section.code}</h4>
+      ) : (
+        ""
+      )}
+      <Attributes
+        instructors={section.instructors}
+        building={splitLocation(section.location).Building}
+        units={convertUnits(section.minUnits, section.maxUnits)}
+        status={section.status}
+        type={section.type}
+        registrationNumber={section.registrationNumber}
+      />
+      {!courseData.sections.every(
+        (section) => section.notes === courseData.sections[0].notes
+      ) && <SectionDescription>{section.notes}</SectionDescription>}
+
+      <DateSection sortedSectionMeetings={sortedSectionMeetings} />
+      <UtilBar>
+        {section.recitations === undefined ||
+        section.recitations.length === 0 ? (
+          <></>
```

Comment:
Use `<React.Fragment />` instead of this or get the opposite of the conditional and do short circuiting.